packages/wallets/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -4,21 +4,23 @@ import './ProgressBar.scss';
 type TProps = {
     is_transition?: boolean;
     active_index: number;
-    indexes: Array<number>;
-    setActiveIndex: React.Dispatch<React.SetStateAction<number>>;
+    indexes: Array<string>;
+    setActiveIndex: (index: string) => void;
 };
 
 const ProgressBar: React.FC<TProps> = ({ is_transition = 'true', active_index, indexes, setActiveIndex }) => {
```

Comment:
@farhan-nurzi-deriv accidentally looks your PR and noticed you are trying to add `string` as a default value for `boolean` :) No problem about typos, but why TS didn't highlight it for you?

Reply:
Hi @sergei-deriv nice catch :). I dont put that value in my PR tho. But i will fix it here. Thanks